cmd/crane/cmd/pull.go

```diff
@@ -92,7 +105,7 @@ func NewCmdPull(options *[]crane.Option) *cobra.Command {
 				}
 				for ref, idx := range indexMap {
 					anns := map[string]string{
-						"dev.ggcr.image.name": ref,
+						"org.opencontainers.image.ref.name": ref,
```

Comment:
This is the expected annotation standardized by image-spec. hence I believe we should follow that. 
also, the other tools follow this convention as well.